project/Project.scala

```diff
@@ -28,37 +28,38 @@ import com.typesafe.sbt.site.SphinxSupport.Sphinx
 object Zipkin extends Build {
   val zipkinVersion = "1.2.0-SNAPSHOT"
 
-  val finagleVersion = "6.26.0"
-  val utilVersion = "6.25.0"
-  val scroogeVersion = "3.19.0"
-  val zookeeperVersions = Map(
+  def finagle(name: String) = "com.twitter" %% ("finagle-" + name) % "6.26.0"
```

Comment:
Correct. I went with this because in quite a few places there's only a single package included, which would mean another round of concatenation / `flatten`.